### DISCUSSION TITLE
Deepstack

### DISCUSSION THREAD

**Discussion by 765pjj** [roflcoopter/viseron#611] [General]
Just a beginners question: 
The deepstack software is NOT included in the docker image like the darknet software, right?

**Comment by roflcoopter** [comment]
That is correct! DeepStack runs as a separate docker container.

DeepStack has been abandoned by its developers tho and I would recommend looking at CodeProject.AI instead which is currently in the dev branch of Viseron.

It is based on DeepStack so it is very similar.
The documentation is here: https://dev--viseron.netlify.app/components-explorer/components/codeprojectai

**Reply by 765pjj** [reply]
Thanks, will try that soon.
